data/lang/ui-core/en.json

```diff
@@ -951,10 +951,26 @@
     "description": "Tooltip for follow mode 'orientation' button",
     "message": "{targetname}: Follow orientation"
   },
+  "HUD_FOLLOW_ORIENTATION_NOT_AVAILABLE": {
+    "description": "Tooltip for follow mode 'orientation' button, when it's not available for the current object.",
+    "message": "{targetname}: Follow orientation is not available."
+  },
+  "HUD_FOLLOW_ORIENTATION_NOT_AVAILABLE_TOO_FAR": {
+    "description": "Tooltip for follow mode 'orientation' button, when it's not available for the current object, because it's too far.",
+    "message": "{targetname}: Follow orientation is not available - too far."
+  },
```

Comment:
These strings are a little bit long for optimal tooltips; I'd almost recommend trying to plumb in support for custom tooltip functions to add multiline/colored tooltip strings (e.g. "{targetname}: Follow Orientation" \<newline\> "Unavailable - Too Far"). That's outside the scope of this PR for right now though, so just something to keep as an idea for future improvements.